docs/source/developing_with_sdk/core_concepts.md

```diff
@@ -38,7 +38,7 @@ To develop and deploy your MONAI App, you can follow the steps below (click a no
 ```{mermaid}
 :alt: Developing Application
 :align: center
-:caption: ⠀⠀Steps to developing application
+:caption: ⠀⠀Steps to develop application
```

Comment:
`application` -> `an application` or `applications`

Reply:
Updated! Thank you!
Also updated

`Steps to package, local-running, and deploying MONAI Application Package (MAP)` to `Steps to package, local-running, and deploying a MONAI Application Package (MAP)` (adding 'a') in other part of the document.